contrib/http_util/src/main/java/io/opencensus/contrib/http/util/HttpStatsUtil.java

```diff
@@ -0,0 +1,435 @@
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_CLIENT_RESPONSE_BYTES =
+      Measure.MeasureLong.create(
+          getName("client", "response_bytes"),
+          "Client-side HTTP response body size (uncompressed)",
+          UNIT_SIZE);
+
+  /**
+   * {@link Measure} for the client-side end-to-end latency.
+   *
+   * @since 0.13
+   */
+  public static final MeasureDouble HTTP_CLIENT_LATENCY =
+      Measure.MeasureDouble.create(
+          getName("client", "latency"), "Client-side end-to-end latency", UNIT_LATENCY);
+
+  /**
+   * {@link Measure} for the number of server-side HTTP requests started.
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_SERVER_REQUEST_COUNT =
+      Measure.MeasureLong.create(
+          getName("server", "request_count"),
+          "Number of server-side HTTP requests started",
+          UNIT_COUNT);
+
+  /**
+   * {@link Measure} for the server-side HTTP request body size if set as ContentLength
+   * (uncompressed).
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_SERVER_REQUEST_BYTES =
+      Measure.MeasureLong.create(
+          getName("server", "request_bytes"),
+          "Server-side HTTP request body size if set as ContentLength (uncompressed)",
+          UNIT_SIZE);
+
+  /**
+   * {@link Measure} for the server-side HTTP response body size (uncompressed).
+   *
+   * @since 0.13
+   */
+  public static final MeasureLong HTTP_SERVER_RESPONSE_BYTES =
+      Measure.MeasureLong.create(
+          getName("server", "response_bytes"),
+          "Server-side HTTP response body size (uncompressed)",
+          UNIT_SIZE);
+
+  /**
+   * {@link Measure} for the server-side end-to-end latency.
+   *
+   * @since 0.13
+   */
+  public static final MeasureDouble HTTP_SERVER_LATENCY =
+      Measure.MeasureDouble.create(
+          getName("server", "latency"), "Server-side end-to-end latency", UNIT_LATENCY);
+
+  /**
+   * {@link TagKey} for the value of the HTTP host header.
+   *
+   * @since 0.13
+   */
+  public static final TagKey HTTP_HOST = TagKey.create("http.host");
+
+  /**
+   * {@link TagKey} for the numeric HTTP response status code. If a transport error occurred and no
+   * status code was read, use "error" as the {@code TagValue}.
+   *
+   * @since 0.13
+   */
+  public static final TagKey HTTP_STATUS_CODE = TagKey.create("http.status");
+
+  /**
+   * {@link TagKey} for the URL path (not including query string) in the request.
+   *
+   * @since 0.13
+   */
+  public static final TagKey HTTP_PATH = TagKey.create("http.path");
+
+  /**
+   * {@link TagKey} for the HTTP method of the request, capitalized (GET, POST, etc.).
+   *
+   * @since 0.13
+   */
+  public static final TagKey HTTP_METHOD = TagKey.create("http.method");
+
+  /**
+   * {@link View} for count of client-side HTTP requests started.
+   *
+   * @since 0.13
+   */
+  public static final View HTTP_CLIENT_REQUEST_COUNT_VIEW =
```

Comment:
Consider splitting HTTP measures and views into two files, since it's likely that other libraries only need to load and record against HTTP measures.

Similar issue to gRPC constants: https://github.com/census-instrumentation/opencensus-java/issues/188.